content/news/buzz/october-2024-status-update/contents.lr

```diff
@@ -0,0 +1,57 @@
+title: October 2024 Status Update
+---
+author: Russell Keith-Magee
+---
+body:
+
+In October, BeeWare saw some important releases, plus more good progress on binary packaging for iOS.
+
+What we've done
+----------------
+
+* Most importantly, we released `Briefcase 0.3.20 <https://pypi.org/project/briefcase/0.3.20/>`__ and `Toga 0.4.8 <https://pypi.org/project/toga/0.4.8/>`__, including support for Python 3.13 - which includes the official support in Python for iOS and Android! This is a major milestone for BeeWare as a project, representing a significant portion of the work done over the last 12 months.
+
+* We've prepared an `initial patch to cibuildwheel that is able to build and test simple iOS wheels <https://github.com/freakboy3742/cibuildwheel/tree/ios-support>`__. This patch isn't ready to submit upstream, but it is able to build simple iOS wheels.
+
+* We've submitted a patch to Pillow to `isolate its build system from Homebrew when building on macOS <https://github.com/python-pillow/Pillow/pull/8497>`__. This is essential for iOS support, as it's easy for Homebrew macOS ARM64 binaries to leak into iOS builds; but it also has benefits for macOS builds.
+
+* We've made `a number of improvements to multibuild <https://github.com/multi-build/multibuild>`__, the tooling that Pillow uses to compile non-Python binary dependencies. 
+
+* We've `modified the CPython iOS testbed project <https://github.com/python/cpython/pull/126169>`__ so that it can be used as a testbed for *any* iOS Python project.
+
+* We've `improved error reporting when Briefcase can't clone a template <https://github.com/beeware/briefcase/pull/2033>`__.
+
+* We've switched to using ``httpx`` instead of ``requests`` for `Briefcase's internal download handling <https://github.com/beeware/briefcase/pull/2041>`__. This provides slightly better error handling, better options for improving HTTP/2 usage, and we're now consistently using ``httpx`` in Briefcase and in our example code.
+
+* We've modified Toga to `lazily loads components <https://github.com/beeware/toga/pull/2686>`__, rather than importing everything into the ``toga`` namespace at startup. This should improve application startup times, especially on platforms like mobile and web where this startup time is noticeable.
```

Comment:
```suggestion
* We've modified Toga to `lazily load components <https://github.com/beeware/toga/pull/2686>`__, rather than importing everything into the ``toga`` namespace at startup. This should improve application startup times, especially on platforms like mobile and web where this startup time is noticeable.
```